packages/actor-init-query/lib/cli/CliArgsHandlerBase.ts

```diff
@@ -185,7 +226,8 @@ export class CliArgsHandlerBase implements ICliArgsHandler {
     if (args.sources.length > 0) {
       context.sources = context.sources || [];
       args.sources.forEach((sourceValue: string) => {
-        const source = CliArgsHandlerBase.getSourceObjectFromString(sourceValue);
+        let sourceValueURI: string = CliArgsHandlerBase.URIfy(sourceValue);
+        const source = CliArgsHandlerBase.getSourceObjectFromString(sourceValueURI);
```

Comment:
The problem with adding this logic to the CLI handler, is that this will not work when the engine is invoked directly via the JS API.

An alternative place to add this would be in https://github.com/comunica/comunica/blob/master/packages/actor-dereference-file/lib/ActorDereferenceFile.ts#L36

Reply:
Ah, that's a good point. My bad going head-first without getting myself familiar with architecture.
Great to see that the same actor is responsible for file dereferencing with no regard where it happens.
Thanks for the pointer.